bin/boilerplate/_extras/figures.md

```diff
@@ -28,8 +28,7 @@ title: Figures
             var h1text = htmlDocArticle.getElementsByTagName("h1")[0].innerHTML;
 
             var htitle = document.createElement('h2');
-            htitle.innerHTML = 'Episode "' + h1text + '"';
-            htitle.style = "margin-top: 100px;";
+            htitle.innerHTML = h1text;
```

Comment:
I would prefer to keep the word `Episode` to differentiate it from figure titles.